### DISCUSSION TITLE
Error saat menjalankan ggplotly

### DISCUSSION THREAD

**Discussion by rizkiajimahardika** [teamalgoritma/community#86] [Q&A] [answered]
semua library sudah di jalankan, namun error saat menjalankan ggplotly.

![error ggplotly](https://user-images.githubusercontent.com/79896604/167278226-c66709e0-07f6-4c0e-9534-5e72d7936a23.png)

**Comment by inytss** [comment, marked as answer]
Dear @rizkiajimahardika ,
Berdasarkan informasi yang Bapak berikan, sepertinya error terjadi dikarenakan Bapak memberikan parameter `tooltip="text"` dimana pada code `ggplot()` Bapak tidak mendefinisikan parameter `text` sama sekali. Boleh dicoba untuk menjalankan plotly nya dengan code `ggplotly(plot1)`.

**Reply by rizkiajimahardika** [reply]
baik kak terima kasih atas jawabannya
